gorm/collection_operators.go

```diff
@@ -82,13 +82,13 @@ func ApplyCollectionOperatorsWithSearchingEx(ctx context.Context, db *gorm.DB, o
 }
 
 // ApplySearchingEx applies searching operator s to gorm instance db.
-func ApplySearchingEx(ctx context.Context, db *gorm.DB, s *query.Searching, obj interface{}, c SearchingConverter) (*gorm.DB, error) {
-	str, args, err := c.SearchingToGorm(ctx, s, obj)
+func ApplySearchingEx(ctx context.Context, db *gorm.DB, s *query.Searching, obj interface{}, fieldsForFTS []string, c SearchingConverter) (*gorm.DB, error) {
+	str, err := c.SearchingToGorm(ctx, s, fieldsForFTS, obj)
 	if err != nil {
 		return nil, err
 	}
 	if str != "" {
-		return db.Where(str, args...), nil
+		return db.Where(str, s.Query), nil
```

Comment:
SearchingToGorm returns only mask. There should be also to_tsvector("+mask+") @@ plainto_tsquery(?) added somewhere.